exchange_rate.py

```diff
@@ -0,0 +1,6 @@
+from exchangeratesapi import Api
+
+api = Api()
+
+# Get the latest foreign exchange rates:
+print(api.get_rates('USD'))
```

Comment:
Let's put this in its own function in `script.py`, similar to the other functions in this repo, tiggered by a command-line argument. Maybe then we can set up a merge conflict for @RainWatcher to solve. 😄